sdgx/metrics/column/jsd.py

```diff
@@ -0,0 +1,118 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.stats import entropy, gaussian_kde
+
+from sdgx.data_process.transform.transform import DataTransformer
+from sdgx.metrics.column.base import ColumnMetric
+
+
+class JSD(ColumnMetric):
+    '''JSD : Jensen Shannon Divergence
+    
+    This class is used to calculate the Jensen Shannon divergence value betweenthe target columns of real data and synthetic data.
+
+    Currently, we support discrete and continuous columns as inputs.
+    '''
+    def __init__(self) -> None:
+        super().__init__()
+        self.lower_bound = 0
+        self.upper_bound = 1
+        self.metric_name = "jensen_shannon_divergence"
+
+    @classmethod
+    def calculate(cls,\
+                  real_data: pd.DataFrame,\
+                  synthetic_data: pd.DataFrame,\
+                  cols: list[str] | None,\
+                  discrete: bool = True,\
+                  ) -> pd.DataFrame:
```

Comment:
```suggestion
    def calculate(
        cls,
        real_data: pd.DataFrame,
        synthetic_data: pd.DataFrame,
        cols: list[str] | None,
        discrete: bool = True,
    ) -> pd.DataFrame:
```

No need to use slash

Reply:
Noticed, parameters in multiple line will not use slash.